R/label-placer.R

```diff
@@ -201,6 +203,46 @@ label_placer_manual <- function(breaks, x, y, theta) {
 }
 
 
+#' @param rot_adjuster Function that standardizes the rotation angles of the labels.
```

Comment:
I believe this argument is already documented and doesn't need to be documented again.

Reply:
Yes it is. I'm going to removed